pkg/output/output.go

```diff
@@ -91,59 +100,82 @@ func NewOutputWithHelmReleases(helmReleases []*release.Release) Output {
 }
 
 // ToFile dispatches a message to file
-func (output Output) ToFile(filename string) error {
+func (output Output) ToFile(filename string, format string) error {
 	output.dedupe()
-	data, err := json.Marshal(output)
-	if err != nil {
-		klog.Errorf("Error marshaling json: %v", err)
-		return err
-	}
+	if format == JSONFormat {
+		data, err := json.Marshal(output)
+		if err != nil {
+			klog.Errorf("Error marshaling json: %v", err)
+			return err
+		}
 
-	err = ioutil.WriteFile(filename, data, 0644)
-	if err != nil {
-		klog.Errorf("Error writing to file %s: %v", filename, err)
+		err = ioutil.WriteFile(filename, data, 0644)
+		if err != nil {
+			klog.Errorf("Error writing to file %s: %v", filename, err)
+		}
+	} else if format == TableFormat {
+		file, err := os.Create("records.csv")
+		defer file.Close()
+		if err != nil {
+			return err
+		}
+		w := csv.NewWriter(file)
+		defer w.Flush()
+		header := []string{"Release Name", "Chart Name", "Namespace", "HelmVersion", "Installed", "Latest", "Old", "Deprecated"}
+		var data [][]string
+		data = append(data, header)
+		for _, rl := range output.HelmReleases {
+			row := []string{rl.ReleaseName, rl.ChartName, rl.Namespace, rl.HelmVersion, rl.Installed.Version, rl.Latest.Version, strconv.FormatBool(rl.IsOld), strconv.FormatBool(rl.Deprecated)}
+			data = append(data, row)
+		}
+		w.WriteAll(data)
```

Comment:
I think this would be a bit better as a switch, that way we can more easily add formats down the road.